packages/react-icons/src/components/OneDriveIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import createSvgIcon from '../utils/createSvgIcon';
+
+const OneDriveIcon = createSvgIcon({
+  svg: ({ classNames }) => (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0,0,2048,2048">
```

Comment:
Should `classNames` get injected on the root?

Reply:
`createSvgIcon` is doind that. I have two alternatives of the factory once that is for injecting only the svg and another one that allows user to have custom props and define the root element too. Let's see if we will need the both or choose one